README.md

```diff
@@ -65,7 +65,7 @@ Adapt the page to the following screens:
 11. `git push origin develop` - to send you code for PR.
 12. Create a Pull Request (PR) from your branch `develop` to branch `master` of original repo.
 13. Replace `<your_account>` with your Github username in the
-  [DEMO LINK](https://<your_account>.github.io/layout_dia/).
+  [DEMO LINK](https://ladrom.github.io/layout_dia/).
```

Comment:
![image](https://user-images.githubusercontent.com/38065883/148056350-15ec48d3-7a90-417a-ba21-d7f9c8e08665.png)
Черный фон должен растягиваться на всю ширину страницы, как в футере

Reply:
![image](https://user-images.githubusercontent.com/38065883/148056459-1e238193-0579-4601-9b66-7243ab08e065.png)
Тут не тот цвет на фоне, который указан в дизайне

Reply:
![image](https://user-images.githubusercontent.com/38065883/148056611-c6d07d3b-9d35-4492-8c58-60610bee224e.png)
Тут тоже не тот цвет

Reply:
https://user-images.githubusercontent.com/38065883/148056744-c7e7e8d4-68fd-47fd-981a-a3c41dda0765.mov

Почему ссылки в меню никуда не ведут?

Reply:
Привет. 

1. По черному фону я видимо неправильно понял вот это задание: on large screens limit the width of black container to 1600px, so that left outer space have a standart grey background color. Хотя, да, в футере ведь не ограничил размер контейнера.
2. С цветом фона вышла ошибка, потому что изначально в файле уже была переменная с похожим серым оттенком, и я как-то недоглядел. При этом меня не покидает чувство, что и #e5e5e5 - неправильный цвет, потому что в фигме бэкграунд похож на слегка голубоватый, ну или это уже у меня экран так передает. 
3. Добавить ссылки в меню просто забыл. Извиняюсь.